src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#index.html"
```

Comment:
The href attribute value '#index.html' is used here, which is intended for linking to an anchor within the same page. If you meant to link to the homepage, consider using 'index.html' without the hash symbol.